hub/src/server/config-schema.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "serverName": { "type": "string" },
+    "port": { "type": "integer" },
+    "driver": { "type": { "enum":
+                          [ "aws", "azure", "disk", "google-cloud" ] } },
+    "bucket": { "type": "string" },
+    "readURL": { "type": "string" },
+    "pageSize": { "type": "integer"},
```

Comment:
This should be a positive integer, and probably not too big (i.e. no bigger than 4096).